guides/source/upgrading_ruby_on_rails.md

```diff
@@ -262,6 +262,22 @@ config.cache_store = :mem_cache_store, "cache.example.com", pool: false
 
 See the [caching with Rails](https://guides.rubyonrails.org/caching_with_rails.html#connection-pool-options) guide for more information.
 
+### `SQLite3Adapter` now configured to be used in a strict mode
+
+The use of a strict mode disables double-quoted string literals.
+
+SQLite has some quirks around double-quoted string literals.
+It first tries to consider double-quoted strings as identifier names, but if they don't exist
+it then considers them as string literals. Because of this, typos can silently go unnoticed.
```

Comment:
What about to link to https://www.sqlite.org/quirks.html#double_quoted_string_literals_are_accepted as well in various places (like here)?